nupic/algorithms/anomaly.py

```diff
@@ -123,25 +126,9 @@ def compute(self, activeColumns, predictedColumns,
                               	(used in anomaly-likelihood)
     @return the computed anomaly score; float 0..1
```

Comment:
@vitaly-krugl about anomalyScore being in range of float 0..1: the docstring should still be valid, documentation for custom functions require the score to be such.

@scottpurdy about weighted mode: the change is from `weighted = raw * (1-likelihood)` to `weighted = raw * likelihood` ...so the performance should not change, only the interpretation, should I revert, and open new issue?